dipy/reconst/tests/test_ivim.py

```diff
@@ -0,0 +1,56 @@
+""" Testing DTI
+
+"""
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from nose.tools import (assert_true, assert_equal,
+                        assert_almost_equal, assert_raises)
+import numpy.testing as npt
+from numpy.testing import (assert_array_equal, assert_array_almost_equal,
+                           assert_)
+import nibabel as nib
+
+import scipy.optimize as opt
+
+from dipy.reconst import ivim as ivim
+from dipy.reconst.ivim import ivim_function
+from dipy.data import get_data, dsi_voxels, get_sphere
+import dipy.core.gradients as grad
+from dipy.sims.voxel import single_tensor
+from dipy.io.gradients import read_bvals_bvecs
+from dipy.core.gradients import (gradient_table, GradientTable,
+                                 gradient_table_from_bvals_bvecs,
+                                 reorient_bvecs)
+from dipy.sims.voxel import multi_tensor
+
+
+def test_nlls_fit():
+    """
+    Test the implementation of NLLS
+    """
+    fimg, fbvals, fbvecs = get_data('small_101D')
+    bvals, bvecs = read_bvals_bvecs(fbvals, fbvecs)
+    gtab = gradient_table(bvals, bvecs)
+
+    f, D_star, D = 90, 0.0015, 0.0008
+
+    mevals = np.array(([D_star, D_star, D_star], [D, D, D]))
+    # This gives an isotropic signal
+
+    signal = multi_tensor(gtab, mevals, snr=None, fractions=[f, 100 - f])
```

Comment:
Ah, yes. I am changing it to stay consistent. In multi_tensor it is taken as 0 - 100. Should I follow that or use 0 - 1 for "f" and while passing to multi_tensor do a ```fractions = [100*f, 100*(1-f)]``` @arokem. What is dipy's take on this ?

@etpeterson Thanks for pointing it out. But even after changing it, the overflow errors persist. Can it be because the data and bvalues not what it is supposed to be for this model. (I am using 'small_101D', I 'll take a look at the data Eric posted)

I think we might need tests to determine the range of f, D_star and D such that results are valid while using this model. Incorrect parameters give a very bad fit.